README.md

```diff
@@ -20,6 +20,37 @@ Via Yarn:
 yarn add react-experimentify
 ```
 
+### Mixpanel
+
+This component will render a control or single variant dependent upon its `showVariant` prop. Additionally, it will signal to mixpanel that an experiment has started only when it is in the user's viewport. This is achieved using [react-intersection-observer](https://github.com/thebuilder/react-intersection-observer). It will only fire once per user/device, based upon [Mixpanel's `distinct_id`](https://help.mixpanel.com/hc/en-us/articles/115004509406-Distinct-IDs-). 
```

Comment:
note: react-intersection-observer requires a polyfill for some browsers (see: https://github.com/thebuilder/react-intersection-observer#polyfill)